### PR TITLE
Add year to home page text

### PR DESCRIPTION
**Pull request by bazzadp**
The text on the home page includes a few stats but doesn't make it clear these stats are only for that year's edition. I think we should make this more clear.

@HTTPArchive/translators I've done my best to translate all the languages but if you get a chance, can you review the changes for your specific languages and let me know if it's good, or suggestion any better language. If I don't hear back before the next release (probably end of the week), then I'll merge and can we open another pull requests if changes are required.

The three changes are I'm proposing are:

**Introduction**

Old:
> It is comprised of 22 chapters spanning aspects of page content, user experience, publishing, and distribution.

New:
> The 2020 edition is comprised of 22 chapters spanning aspects of page content, user experience, publishing, and distribution.

**Contributors**

Old:
> 124 people have volunteered countless hours in the planning, research, writing and production phases.

New:
> 124 people have volunteered countless hours in the planning, research, writing and production phases of the 2020 Web Almanac.

**Methodology:**

Old:
> Unless otherwise noted, the metrics in all of the 22 chapters of the Web Almanac are sourced from... The August 2020 dataset was used as the basis for the Web Almanac’s metrics.

New:
> Unless otherwise noted, the metrics in all of the 22 chapters of the 2020 Web Almanac are sourced from... The August 2020 dataset was used as the basis for the 2020 Web Almanac’s metrics.

### CODE REVIEW

**Review by noah-vdv — APPROVED** [review]
LGTM for the dutch version

File: src/templates/zh-TW/base.html [hunk 1]
```diff
@@ -109,15 +109,15 @@
 {% endif %}
 
 {% block contributors_description %}
-Web Almanac 網路年鑑是被熱情的網路社群擁抱而茁壯。 {{ self.contributors() }} 位團隊成員志願挹注難以計數的泉源在規劃、研究、撰寫、發佈的各個階段。
+Web Almanac 網路年鑑是被熱情的網路社群擁抱而茁壯。 {{ self.contributors() }}位團隊{{ year }} Web Almanac成員志願挹注難以計數的泉源在規劃、研究、撰寫、發佈的各個階段。
```
Comment (CYBAI) [inline src/templates/zh-TW/base.html:112]:
```suggestion
Web Almanac 網路年鑑是被熱情的網路社群擁抱而茁壯。 {{ self.contributors() }} 位團隊成員志願挹注難以計數的泉源在規劃、研究、撰寫、發佈 {{ year }} Web Almanac 的各個階段。
```

**Review by CYBAI — APPROVED** [review]
LGTM with the suggestion fixed 🙏

File: src/templates/uk/base.html [hunk 2]
```diff
@@ -109,15 +109,15 @@
 {% endif %}
 
 {% block contributors_description %}
-Створення Web Almanac стало можливим завдяки важкій роботі веб-спільноти. {{ self.contributors() }} надзвичайних людей вклали безліч години у планування, дослідження, написання та продакшен.
+Створення Web Almanac стало можливим завдяки важкій роботі веб-спільноти. {{ self.contributors() }} надзвичайних людей вклали безліч години у планування, дослідження, написання та продакшен Веб-альманаху {{ year }} року.
```
Comment (tymosh) [inline src/templates/uk/base.html:112]:
```suggestion
Створення Web Almanac стало можливим завдяки важкій роботі веб-спільноти. {{ self.contributors() }} надзвичайних людей вклали безліч години у планування, дослідження, написання та продакшен Web Almanac {{ year }}.
```

**Review by tymosh — APPROVED** [review]
Ukrainian looks cool!

File: src/templates/fr/base.html [hunk 3]
```diff
@@ -109,15 +109,15 @@
 {% endif %}
 
 {% block contributors_description %}
-Le Web Almanac a été rendu possible grâce au travail acharné de la communauté Web. {{ self.contributors() }} personnes ont consacré bénévolement d’innombrables heures à le planifier, faire des recherches, le rédiger et le mettre à votre disposition.
+Le Web Almanac a été rendu possible grâce au travail acharné de la communauté Web. {{ self.contributors() }} personnes ont consacré bénévolement d’innombrables heures à le planifier, faire des recherches, le rédiger et le mettre à votre disposition du Web Almanac {{ year }}.
```
Comment (stephanebachelier) [inline src/templates/fr/base.html:112]: @bazzadp I would change the last sentence for `{{ self.contributors() }} personnes ont consacré bénévolement d’innombrables heures à planifier, faire des recherches, rédiger et mettre à votre disposition l'édition {{ year }} du Web Almanac.`

Comment (bazzadp) [reply src/templates/fr/base.html:112]:
```suggestion
Le Web Almanac a été rendu possible grâce au travail acharné de la communauté Web. {{ self.contributors() }} personnes ont consacré bénévolement d’innombrables heures à planifier, faire des recherches, rédiger et mettre à votre disposition l'édition {{ year }} du Web Almanac.
```

Comment (bazzadp) [reply src/templates/fr/base.html]: Thanks @stephanebachelier - changed!

File: src/templates/pt/base.html [hunk 4]
```diff
@@ -109,15 +109,15 @@
 {% endif %}
 
 {% block contributors_description %}
-O Web Almanac Isso foi possível graças ao trabalho árduo da comunidade da web. {{ self.contributors() }} as pessoas ofereceram inúmeras horas nas fases de planejamento, pesquisa, redação e produção.
+O Web Almanac Isso foi possível graças ao trabalho árduo da comunidade da web. {{ self.contributors() }} as pessoas ofereceram inúmeras horas nas fases de planejamento, pesquisa, redação e produção do Web Almanac {{ year }}.
```
Comment (emanuelgsouza) [inline src/templates/pt/base.html:112]: You could change that sentence to "O Web Almanac só foi possível graças ao trabalho árduo da comunidade web. {{ self.contributors() }} pessoas ofereceram inúmeras horas nas fases de planejamento, pesquisa, redação e produção do Web Almanac {{ year }}."

File: src/templates/pt/base.html [hunk 5]
```diff
@@ -20,7 +20,7 @@
 
 {% block mission %}
 <p>
-Nossa missão é combinar as estatísticas brutas e as tendências do HTTP Archive com a experiência da comunidade da web. O Web Almanac é um relatório abrangente sobre o estado da Web, apoiado por dados reais e especialistas confiáveis da Web. É composto por {{ supported_chapters | length }} capítulos que abrangem aspectos do conteúdo da página, experiência do usuário, publicação e distribuição.
+Nossa missão é combinar as estatísticas brutas e as tendências do HTTP Archive com a experiência da comunidade da web. O Web Almanac é um relatório abrangente sobre o estado da Web, apoiado por dados reais e especialistas confiáveis da Web. A edição de {{ year }} é composta por {{ supported_chapters | length }} capítulos que abrangem aspectos do conteúdo da página, experiência do usuário, publicação e distribuição.
```
Comment (emanuelgsouza) [inline src/templates/pt/base.html:23]: Reading becomes more fluid using "comunidade web" instead of "comunidade da web".

**Review by emanuelgsouza — CHANGES_REQUESTED** [review]
If you allowed me, I requested some changes in the Portuguese translation.

File: src/templates/pt/base.html (hunk 4 above)
Comment (bazzadp) [reply src/templates/pt/base.html:112]: Sounds good. Adding as suggestion:

```suggestion
O Web Almanac só foi possível graças ao trabalho árduo da comunidade web. {{ self.contributors() }} pessoas ofereceram inúmeras horas nas fases de planejamento, pesquisa, redação e produção do Web Almanac {{ year }}.
```

File: src/templates/pt/base.html (hunk 5 above)
Comment (bazzadp) [reply src/templates/pt/base.html:23]:
```suggestion
Nossa missão é combinar as estatísticas brutas e as tendências do HTTP Archive com a experiência da comunidade web. O Web Almanac é um relatório abrangente sobre o estado da Web, apoiado por dados reais e especialistas confiáveis da Web. A edição de {{ year }} é composta por {{ supported_chapters | length }} capítulos que abrangem aspectos do conteúdo da página, experiência do usuário, publicação e distribuição.
```

**Review by emanuelgsouza — APPROVED** [review]

**Review by dtikhonov — COMMENTED** [review]
The Russian portion looks good!  (Sorry for the late review.)

**Pull request merged**